services/elasticloadbalancing/src/main/resources/codegen-resources/customization.config

```diff
@@ -7,5 +7,13 @@
         "DuplicateAccessPointNameException" : "DuplicateLoadBalancerNameException",
         "TooManyAccessPointsException" : "TooManyLoadBalancersException",
         "InvalidEndPointException" : "InvalidInstanceException"
+    },
+    "marshallAutoConstructListMembers": {
```

Comment:
ELB V2 may need similar customizations.

Reply:
Per above comment, removing this customization (looks like it wasn't present in v2).